src/Avalonia.Controls/Flyouts/PopupFlyoutBase.cs

```diff
@@ -47,6 +47,13 @@ public abstract class PopupFlyoutBase : FlyoutBase, IPopupHostProvider
         public static readonly StyledProperty<IInputElement?> OverlayInputPassThroughElementProperty =
             Popup.OverlayInputPassThroughElementProperty.AddOwner<PopupFlyoutBase>();
         
+        /// <summary>
+        /// Defines the <see cref="OverlayInputPassThroughElement"/> property
```

Comment:
Missing period (that will raise warnings in some code analyzers).

Reply:
🤔 We have no such period in other comments in this class. So I don't know what to do here best. 
- Add the period for this property only?
- Add the period everywhere introducing a change to unrelated areas? 
- Address it in a separate issue / PR to fix the warnings everywhere?

For now I'll leave it as is for consistence

Reply:
Its usually good to update the entire file then rather than just let it be forgotten. These types of things can be updated as each file is modified and issues are noticed.

As stated elsewhere we really do need formatting checks on PRs and enable analyzers with editor configs.